package.json

```diff
@@ -10,8 +10,8 @@
     "flow": "flow",
     "lint": "eslint .",
     "test": "ava --verbose 'lib/**/*.test.js'",
-    "coverage": "nyc yarn test && nyc report --reporter=text-lcov | coveralls",
-    "release": "yarn build && lerna publish --conventional-commits"
+    "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
```

Comment:
Wow, I was just trying to reproduce the problem and it was a really tricky bug :D Thanks for resolving this!